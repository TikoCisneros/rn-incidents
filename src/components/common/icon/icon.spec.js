import React from 'react';
import { shallow } from 'enzyme';
import Icon from './icon';

describe('Icon component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Icon name="person" />);
    expect(wrapper).toMatchSnapshot();
  });
});