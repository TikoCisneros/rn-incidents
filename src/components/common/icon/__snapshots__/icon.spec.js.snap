// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component renders correctly 1`] = `
<Icon
  allowFontScaling={false}
  name="person"
  size={12}
  style={
    Array [
      Object {
        "fontSize": 20,
      },
      undefined,
    ]
  }
/>
`;
